cupy/cuda/cub.pyx

```diff
@@ -103,11 +103,16 @@ def device_reduce(ndarray x, int op, out=None, bint keepdims=False):
         raise ValueError(
             "output parameter for reduction operation has the wrong number of "
             "dimensions")
-    if op < 0 or op > 2:
-        raise ValueError("only CUPY_CUB_SUM, CUPY_CUB_MIN, and CUPY_CUB_MAX "
-                         "are supported.")
+    if op < 0 or op > 4:
```

Comment:
Would it be possible to use the mnemonics defined in the .pxd instead of raw numbers?

Reply:
Not sure if it improves readability or not, but it's definitely doable. Let me change it.

Reply:
Done. PTAL.